.github/integration/tests/sda/60_api_admin_test.sh

```diff
@@ -1,10 +1,131 @@
 #!/bin/sh
 set -e
+cd shared || true
 
 token="$(curl http://oidc:8080/tokens | jq -r '.[0]')"
+# Upload a file and make sure it's listed
 result="$(curl -sk -L "http://api:8080/users/test@dummy.org/files" -H "Authorization: Bearer $token" | jq '. | length')"
 if [ "$result" -ne 2 ]; then
     echo "wrong number of files returned for user test@dummy.org"
-    echo "expected 4 got $result"
+    echo "expected 2 got $result"
     exit 1
+fi
+
+
+# Reupload a file under a different name, test to delete it
+s3cmd -c s3cfg put "NA12878.bam.c4gh" s3://test_dummy.org/NC12878.bam.c4gh
+
+echo "waiting for upload to complete"
+URI=http://rabbitmq:15672
+if [ -n "$PGSSLCERT" ]; then
+   URI=https://rabbitmq:15671
+fi
+RETRY_TIMES=0
+until [ "$(curl -s -k -u guest:guest $URI/api/queues/sda/inbox | jq -r '."messages_ready"')" -eq 4 ]; do
+   echo "waiting for upload to complete"
+   RETRY_TIMES=$((RETRY_TIMES + 1))
+   if [ "$RETRY_TIMES" -eq 30 ]; then
+      echo "::error::Time out while waiting for upload to complete"
+      exit 1
+   fi
+   sleep 2
+done
+
+# get the fileId of the new file
+fileid="$(curl -k -L -H "Authorization: Bearer $token" -H "Content-Type: application/json" "http://api:8080/users/test@dummy.org/files" | jq -r '.[] | select(.inboxPath == "test_dummy.org/NC12878.bam.c4gh") | .fileID')"
+
+output=$(s3cmd -c s3cfg ls s3://test_dummy.org/NC12878.bam.c4gh 2>/dev/null)
+if [ -z "$output" ] ; then
+    echo "Uploaded file not in inbox"
+    exit 1
+fi
+# delete it
+resp="$(curl -s -k -L -o /dev/null -w "%{http_code}\n" -H "Authorization: Bearer $token" -H "Content-Type: application/json" -X DELETE "http://api:8080/file/test@dummy.org/$fileid")"
```

Comment:
```suggestion
resp="$(curl -s -k -L -o /dev/null -w "%{http_code}\n" -H "Authorization: Bearer $token" -X DELETE "http://api:8080/file/test@dummy.org/$fileid")"
```
Content type header not needed since we are not sending any payload.